internal/consensus/common_test.go

```diff
@@ -69,6 +69,10 @@ func configSetup(t *testing.T) *config.Config {
 	require.NoError(t, err)
 	t.Cleanup(func() { os.RemoveAll(configByzantineTest.RootDir) })
 
+	walDir := filepath.Dir(cfg.Consensus.WalFile())
+	ensureDir(t, walDir, 0700)
+	t.Cleanup(func() { os.RemoveAll(walDir) })
```

Comment:
if the dir comes from t.TempDir then I think we don't need to delete it? (I might be wrong about that?)

Reply:
Yes, you're right, fix'd.